WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -72,7 +72,7 @@
 
         <% if @competition.base_entry_fee %>
           <%= alert :info do %>
-            <%= t 'registrations.entry_fee' %><strong><%= humanized_money_with_symbol @competition.base_entry_fee %></strong>
+            <%= t 'registrations.entry_fee', fee: humanized_money_with_symbol @competition.base_entry_fee %>
```

Comment:
I think we flew too close to the sun without parens here. Travis is freaking out: https://travis-ci.org/thewca/worldcubeassociation.org/builds/196541580#L1897-L1906.

I suspect the fix is as simple as adding parens to the call to `humanized_money_with_symbol`, like this:

```erb
<%= t 'registrations.entry_fee', fee: humanized_money_with_symbol(@competition.base_entry_fee) %>
```